core/src/main/java/org/infinispan/metrics/impl/ApplicationMetricsRegistry.java

```diff
@@ -56,6 +61,30 @@ protected MetricRegistry makeRegistry() {
       return MetricRegistries.get(MetricRegistry.Type.APPLICATION);
    }
 
+   @Start
+   public void start() {
+      jmxRegistrar = new JmxRegistrar();
+      try {
+         jmxRegistrar.init();
+      } catch (Exception e) {
+         //throw new IllegalStateException("Failed to initialize base and vendor metrics from JMX MBeans", e);
```

Comment:
I assume that this should be at least a WARN?

Reply:
Oops. No, it should be an ERROR and I wonder why I commented it out. Will fix asap.